trk/src/Screens/ClimbDetail/index.js

```diff
@@ -277,14 +277,21 @@ function ClimbDetail(props) {
               <View style={styles.climbPhoto}>
                 {climbImageUrl ? <Image source={{ uri: climbImageUrl }} style={{ width: '100%', height: '100%' }} /> : <Text>Loading...</Text>}
               </View>
+             
               <View style={{ marginTop: 20 }}>
                 <View style={styles.button}>
-                <Button title='Review this climb' onPress={onFeedback} />
+                  <Button title='Review this climb' onPress={onFeedback} />
                 </View>
                 <View style={styles.button}>
-                <Button title='Share' onPress={onShare} />
+                  <Button title='Share' onPress={onShare} />
                 </View>
               </View>
+              {climbData.info && (
+                <View style={styles.infoBox}>
+                  <Text style={styles.subheading}>Climb Info</Text>
+                  <Text style= {styles.info}>{climbData.info}</Text>
+                  </View>
+              )}
             </View>
```

Comment:
added the climbinfo (if the field in the climb document exists, otherwise it's not displayed at all)